### PR TITLE
Change assemby name to 'crc-tray'

### PR DESCRIPTION
**Pull request by anjannath**
~Use fody costura package to embed all the references in the assembly, this will generate a single binary for the tray with all the dll of the references embedded as resources~

- do not creating debug symbols for release builds
- change the assembly name to expected `crc-tray`

### CODE REVIEW

File: crc-tray.csproj [hunk 1]
```diff
@@ -75,34 +76,162 @@
...
+    <Reference Include="Costura, Version=5.3.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>packages\Costura.Fody.5.3.0\lib\netstandard1.0\Costura.dll</HintPath>
+    </Reference>
     <Reference Include="Microsoft.Bcl.AsyncInterfaces, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
       <HintPath>packages\Microsoft.Bcl.AsyncInterfaces.5.0.0\lib\net461\Microsoft.Bcl.AsyncInterfaces.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Win32.Primitives, Version=4.0.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>packages\Microsoft.Win32.Primitives.4.3.0\lib\net46\Microsoft.Win32.Primitives.dll</HintPath>
+      <Private>True</Private>
+      <Private>True</Private>
```
Comment (gbraad) [inline crc-tray.csproj:88]: Private, Private? Double privacy :-P ?

**Review by gbraad — CHANGES_REQUESTED** [review]
A lot of duplication in the `Reference` sections

File: crc-tray.csproj (hunk 1 above)
Comment (anjannath) [reply crc-tray.csproj:88]: Auto generated :P will clean this up now..

**Comment by anjannath** [comment]
builds from the CI doesn't work :( complain about missing `System.Text.Encoding.Web`

**Review by gbraad — APPROVED** [review]

**Comment by gbraad** [comment]
Related to: code-ready/crc#2090

**Pull request merged**